src/Nethermind/Nethermind.Merge.Plugin/IMergeConfig.cs

```diff
@@ -42,5 +43,17 @@ public interface IMergeConfig : IConfig
 
         [ConfigItem(Description = "URL to Builder Relay. If set when building blocks nethermind will send them to the relay.", DefaultValue = "null")]
         string? BuilderRelayUrl { get; set; }
+
+        [ConfigItem(Description = "Reduces block EngineApi latency by disabling Garbage Collection during EngineApi calls.", DefaultValue = "true")]
+        public bool PrioritizeBlockLatency { get; set; }
+
+        [ConfigItem(Description = "Reduces memory usage by forcing Garbage Collection between EngineApi calls. Accept values `NoGc` (-1), Gen0 (0), Gen1 (1), Gen2 (2).", DefaultValue = "Gen1")]
+        public GcLevel SweepMemory { get; set; }
+
+        [ConfigItem(Description = "Reduces process used memory. Accept values `No` which disables it, `Yes` which compacts normal heaps, `Full` compacts large object heap too (only when SweepMemory is set to `Gen2`).", DefaultValue = "Yes")]
```

Comment:
@benaadams @LukaszRozmej cosmetic thing. Do you think that "Merge" section is the best for having such config options?

Reply:
Probably should be in a GC section? 🤔

Reply:
So the answer is yes and no. No it is not the best as there is little connection between merge and GC by itself. Yes because what we are optimizing is based on EngineAPI and it is not applicable anything outside the merge.

So it is little confusing we have this settings under merge, but if we had them elsewhere it would be confusing for users on some other networks.